charts/datadog-operator/templates/NOTES.txt

```diff
@@ -40,6 +40,22 @@ The maximumGoroutines parameter isn't supported by the Operator 1.0.0-rc.12 and
 Setting a value will not change the default defined in the Operator.
     {{- end }}
 {{- end }}
+
+{{- if (semverCompare ">=1.0.0" .Values.image.tag) }}
+    {{- if .Values.datadogCRDs.migration.datadogAgents.conversionWebhook.enabled }}
+##############################################################################
+####               WARNING: v1alpha1 and conversion webhook deprecation.  ####
+##############################################################################
+
+The DatadogAgent `v1alpha1` reconciliation in the Operator is deprecated since v1.2.0+ and will be removed in v1.7.0.
```

Comment:
```suggestion
DatadogAgent v1alpha1 reconciliation in the Operator is deprecated since v1.2.0+ and will be removed in v1.7.0.
```
I'm leaning towards removing the backticks around "v1alpha1" here and on line 53, but don't feel too strongly